ci/kind/test-netpol-v2-conformance-kind.sh

```diff
@@ -0,0 +1,181 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The script runs NetworkPolicy conformance tests on Kind with different traffic encapsulation modes.
+
+set -eo pipefail
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+_usage="Usage: $0 [--api-version <version>] [--encap-mode <mode>] [--ip-family <v4|v6>] [--help|-h]
+        --api-version                 Set specific network-policy-api version for testing.
+        --encap-mode                  Traffic encapsulation mode. (default is 'encap').
+        --ip-family                   Configures the ipFamily for the KinD cluster.
+        --feature-gates               A comma-separated list of key=value pairs that describe feature gates, e.g. AntreaProxy=true,Egress=false.
+        --setup-only                  Only perform setting up the cluster and run test.
+        --cleanup-only                Only perform cleaning up the cluster.
+        --test-only                   Only run test on current cluster. Not set up/clean up the cluster.
+        --help, -h                    Print this message and exit.
+"
+
+function print_usage {
+    echoerr -n "$_usage"
+}
+
+TESTBED_CMD=$(dirname $0)"/kind-setup.sh"
+YML_CMD=$(dirname $0)"/../../hack/generate-manifest.sh"
+
+function quit {
+  result=$?
+  if [[ $setup_only || $test_only ]]; then
+    exit $result
+  fi
+  echoerr "Cleaning testbed"
+  $TESTBED_CMD destroy kind
+}
+
+api_version="v0.1.0"
+mode=""
+ipfamily="v4"
+feature_gates="AdminNetworkPolicy=true"
+setup_only=false
+cleanup_only=false
+test_only=false
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    --api-version)
+    api_version="$2"
+    shift 2
+    ;;
+    --feature-gates)
+    feature_gates="$2"
+    shift 2
+    ;;
+    --ip-family)
+    ipfamily="$2"
+    shift 2
+    ;;
+    --encap-mode)
+    mode="$2"
+    shift 2
+    ;;
+    --setup-only)
+    setup_only=true
+    shift
+    ;;
+    --cleanup-only)
+    cleanup_only=true
+    shift
+    ;;
+    --test-only)
+    test_only=true
+    shift
+    ;;
+    -h|--help)
+    print_usage
+    exit 0
+    ;;
+    *)    # unknown option
+    echoerr "Unknown option $1"
+    exit 1
+    ;;
+esac
+done
+
+if [[ $cleanup_only == "true" ]];then
+  $TESTBED_CMD destroy kind
+  exit 0
+fi
+
+trap "quit" INT EXIT
+
+manifest_args="$manifest_args --verbose-log --coverage"
+if [ -n "$feature_gates" ]; then
+  manifest_args="$manifest_args --feature-gates $feature_gates"
+fi
+
+IMAGE_LIST=("registry.k8s.io/e2e-test-images/agnhost:2.43" \
+            "antrea/antrea-ubuntu-coverage:latest")
```

Comment:
I think you should use `antrea/antrea-ubuntu:latest` here.
You tag the image in the Github workflow: `docker tag antrea/antrea-ubuntu-coverage:latest antrea/antrea-ubuntu:latest`

You should also make sure that the image is *not* pulled. Actually there is a similar bug in the original script (`ci/kind/test-e2e-kind.sh`), which I will look into...

Reply:
Done